pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
You are using older version of pnpm, the one required for Knossos is 8.6.1 with lock file version 6.1. I'd recommend checking out [corepack](https://nodejs.org/api/corepack.html) to always use the correct version for a project you are working on.